---
title: "RECOMB-CG program"
permalink: /program/
layout: single
---

<table>
      <tr>
          <th>Session #</th>
          <th>Start Time</th>
          <th>End Time</th>
          <th>Type</th>
          <th>Authors(Speaker)</th>
          <th>Title</th>
          <th>Session Chair</th>
      </tr>
      <tr>
          <td> </td>
          <td>08:30</td>
          <td>09:30</td>
          <td>Registration</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
        <td>1</td>
        <td>09:30</td>
        <td>10:30</td>
        <td>Keynote Talk 1</td>
        <td>Alexander Urban, Stanford University</td>
        <td>TBD</td>
        <td>Giltae Song</td>
      </tr>
      <tr>
          <td> </td>
          <td>10:30</td>
          <td>10:45</td>
          <td>Coffee break</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td>2</td>
          <td>10:45</td>
          <td>11:10</td>
          <td>Proceeding talk 1</td>
          <td>Ofek Shami-Schnitzer and Tamir Tuller</td>
          <td>Novel Driver Mutations in GCB Lymphoma Patients That Affect Transcription Factors Binding</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>11:10</td>
          <td>11:35</td>
          <td>Proceeding talk 2</td>
          <td>Yushu Liu and Luay Nakhleh</td>
          <td>Inferring Phylogenetic Trees of Cancer Evolution from Longitudinal Single-Cell Copy Number Profiles</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>11:35</td>
          <td>12:00</td>
          <td>Proceeding talk 3</td>
          <td>Songdi Hu, Vladimir Reinharz and Olivier Tremblay-Savard</td>
          <td>Tree decomposition for reconstructing ancestral RNA sequences of multiple families</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>12:00</td>
          <td>13:30</td>
          <td>Lunch break</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td>3</td>
          <td>13:30</td>
          <td>13:55</td>
          <td>Proceeding talk 4</td>
          <td>Reza Kalhor, Manuel Lafond and Celine Scornavacca</td>
          <td>Whole-genome duplication detection with phylogenomics reconciliation: a scalable approach</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>13:55</td>
          <td>14:20</td>
          <td>Proceeding talk 5</td>
          <td>5.	Alitzel López Sánchez, Guillaume E. Scholz, Peter F. Stadler and Manuel Lafond</td>
          <td>A Sankoff-Rousseau-like Algorithm for Minimizing Lateral Gene Transfers and Losses on Single Origin Characters</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>14:20</td>
          <td>14:45</td>
          <td>Proceeding talk 6</td>
          <td>Yongze Yin, Bryce Kille, Huw Ogilvie, Todd Treangen and Luay Nakhleh</td>
          <td>tMHG-Finder: Tree-guided Maximal Homologous Group Finder for Bacterial Genomes</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>14:45</td>
          <td>15:10</td>
          <td>Proceeding talk 7</td>
          <td>Leo van Iersel, Mark Jones, Jannik Schestag, Celine Scornavacca and Mathias Weller</td>
          <td>Phylogenetic Network Diversity Parameterized by Reticulation Number and Beyond</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>15:10</td>
          <td>15:35</td>
          <td>Proceeding talk 8</td>
          <td>Tanjeem Azwad Zaman, Rabib Jahin Ibn Momin and Md. Shamsuzzoha Bayzid</td>
          <td>On the robustness to gene tree rooting (or lack thereof) of triplet-based species tree estimation methods</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>15:35</td>
          <td>15:45</td>
          <td>Coffee break</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td>4</td>
          <td>15:45</td>
          <td>16:10</td>
          <td>Proceeding talk 9</td>
          <td>Hao Yu and Louxin Zhang</td>
          <td>Exact Counts of Binary Phylogenetic Networks with Two and Three Reticulation Events</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>16:10</td>
          <td>16:35</td>
          <td>Proceeding talk 10</td>
          <td>Xintong Zhou and David Sankoff</td>
          <td>Ancestral pangenomes and their phylogenetic reconstruction</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>16:35</td>
          <td>17:00</td>
          <td>Proceeding talk 11</td>
          <td>Navid Bin Hasan, Sohaib and Md. Shamsuzzoha Bayzid</td>
          <td>QT-WEAVER: Correcting quartet distribution improves phylogenomic analyses despite gene tree estimation error</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>17:00</td>
          <td>17:25</td>
          <td>Proceeding talk 12</td>
          <td>Kaari Landry and Olivier Tremblay-Savard</td>
          <td>Fast calculation of cherry distance on level-1 orchard networks: optimization, heuristic and implementation</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>17:25</td>
          <td>17:30</td>
          <td>Closing remark</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>17:30</td>
          <td>19:30</td>
          <td>Poster session I & Satellites Reception</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td> </td>
          <td> </td>
          <td> </td>
          <td> </td>
          <td> </td>
          <td> </td>
      <tr>
          <td> </td>
          <td>08:30</td>
          <td>09:30</td>
          <td>Registration</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td>5</td>
          <td>09:30</td>
          <td>10:30</td>
          <td>Keynote Talk II</td>
          <td>Choongwon Jeong, Seoul National University</td>
          <td>TBD</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>10:30</td>
          <td>10:45</td>
          <td>Coffee break</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td>6</td>
          <td>10:45</td>
          <td>11:10</td>
          <td>Proceeding talk 13</td>
          <td>Meijun Gao and Kevin Liu</td>
          <td>Detecting and Mapping Local Model Violations During Biomolecular Sequence Analysis: a RE sampling and Visual EvALuation Approach</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>11:10</td>
          <td>11:35</td>
          <td>Proceeding talk 14</td>
          <td>Martin Frith</td>
          <td>A simple way to find related sequences with position-specific probabilities</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>11:35</td>
          <td>12:00</td>
          <td>Proceeding talk 15</td>
          <td>Sarwan Ali, Taslim Murad, Prakash Chourasia, Haris Mansoor, Imdadullah Khan, Pin-Yu Chen and Murray Patterson</td>
          <td>Position Specific Scoring Is All You Need? Revisiting Protein Sequence Classification Tasks</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>12:00</td>
          <td>13:30</td>
          <td>Lunch break</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td>7</td>
          <td>13:30</td>
          <td>14:20</td>
          <td>Keynote Talk III</td>
          <td>Eun-kyeong Jo, Chungnam National University</td>
          <td>When infection biology meets bioinformatics</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>14:20</td>
          <td>14:45</td>
          <td>Proceeding talk 16</td>
          <td>Siyu Chen and David Sankoff</td>
          <td>Residual immunity and seasonality of an epidemic</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>14:45</td>
          <td>15:10</td>
          <td>Proceeding talk 17</td>
          <td>Eric Chen, Tara Newman, John Tyson, Anthea Lam, Michael Chan, Agatha Jassem, Natalie Prystajecky, Shannon Russell and James Zlosnik</td>
          <td>Adapting the cov2clusters tool for clustering MPOXV whole genome sequences</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>15:10</td>
          <td>15:35</td>
          <td>Proceeding talk 18</td>
          <td>Muyao Huang and Martin Frith</td>
          <td>Probability-based sequence comparison finds the oldest ever nuclear mitochondrial DNA segments in mammalian genomes</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>15:35</td>
          <td>15:45</td>
          <td>Coffee break</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td>8</td>
          <td>15:45</td>
          <td>16:10</td>
          <td>Proceeding talk 19</td>
          <td>Triveni Shelke, Vanika Gupta and Ishaan Gupta</td>
          <td>Unraveling Insect Immunity: A Cross-Order Comparative Genomic Analysis of Key Immune Proteins</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>16:10</td>
          <td>16:35</td>
          <td>Proceeding talk 20</td>
          <td>Yue Zhang</td>
          <td>Analyzing Sequence Similarity Distributions in Salmonidae: A Branching Process Approach</td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>16:35</td>
          <td>17:00</td>
          <td>Proceeding talk 21</td>
          <td>Katia Aviña-Padilla, Octavio Zambada-Moreno, Manuel A. Barrios-Izás, Michelle Bustamante-Castillo and Maribel Hernández-Rosales</td>
          <td>Evolutionary Reconstruction of Hormone-bHLH Regulatory Networks in Solanaceae: Phylogenomic Insights from PSTVd-Tomato Interactions</td>
          <td></td>
      </tr>
      <tr>
          <td> </td>
          <td>17:00</td>
          <td>17:05</td>
          <td>Closing remark</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
      <tr>
          <td> </td>
          <td>17:05</td>
          <td>19:30</td>
          <td>Poster session II & Welcome Reception</td>
          <td> </td>
          <td> </td>
          <td> </td>
      </tr>
</table>



